daprdocs/content/en/reference/cli/dapr-init.md

```diff
@@ -59,6 +60,12 @@ You can also specify a specific runtime version. Be default, the latest version
 dapr init --runtime-version 1.4.0
 ```
 
+You can also install Dapr with a particular image variant, for example: `mariner`.
```

Comment:
can you make Mariner a reference link here to these docs https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/#using-mariner-based-images

Reply:
@shivamkm07 - This change is for the v1.9 release, yes? If so, can you target this docs for v1.9 branch?

Reply:
Mariner reference added, Thanks!